# Generated by Django 4.0.4 on 2022-04-14 17:54

import api.models
import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('api', '0008_alter_sobreviventes_latitude_and_more'),
    ]

    operations = [
        migrations.AlterField(
            model_name='inventario',
            name='agua',
            field=models.IntegerField(blank=0, null=0, validators=[api.models.maior_que_zero]),
        ),
        migrations.AlterField(
            model_name='inventario',
            name='alimentacao',
            field=models.IntegerField(blank=0, null=0, validators=[api.models.maior_que_zero]),
        ),
        migrations.AlterField(
            model_name='inventario',
            name='medicacao',
            field=models.IntegerField(blank=0, null=0, validators=[api.models.maior_que_zero]),
        ),
        migrations.AlterField(
            model_name='inventario',
            name='municao',
            field=models.IntegerField(blank=0, null=0, validators=[api.models.maior_que_zero]),
        ),
        migrations.AlterField(
            model_name='sobreviventes',
            name='latitude',
            field=models.FloatField(error_messages={'required': 'Varia de 90 a -90.'}, validators=[django.core.validators.MaxValueValidator(90), django.core.validators.MinValueValidator(-90)]),
        ),
        migrations.AlterField(
            model_name='sobreviventes',
            name='longitude',
            field=models.FloatField(error_messages={'required': 'Varia de 180 a -180.'}, validators=[django.core.validators.MaxValueValidator(180), django.core.validators.MinValueValidator(-180)]),
        ),
    ]
